docs/transactions.md

```diff
@@ -1173,16 +1198,6 @@ already present on the ledger.
     - the existing certificate must be NOC non-root certificate
     - the sender's VID must match the vid field of the existing certificates.
   - the signature (self-signature) and expiration date must be valid.
-- Parameters:
-  - cert: `string` - The NOC non-root Certificate, encoded in X.509v3 PEM format. Can be a PEM string or a file path.
-- In State:
-  - `pki/ApprovedCertificates/value/<Subject>/<SubjectKeyID>`
-  - `pki/ApprovedCertificatesBySubject/value/<Subject>`
-  - `pki/ApprovedCertificatesBySubjectKeyID/value/<SubjectKeyID>`
-  - `pki/NOCCertificates/value/<VID>`
-  - `pki/ChildCertificates/value/<Certificate's Subject>/<Certificate's Subject Key ID>`
-- CLI Command:
-  - `dcld tx pki add-noc-x509-cert --certificate=<string-or-path> --from=<account>`
 
```

Comment:
There are no "non root" NOC certificates. NOC == Root. Intermediate must be called ICA. Please fix everywhere in the doc.

Reply:
This will be implemented in a separate pull request.